etc/picongpu/pizdaint-cscs/picongpu.profile.example

```diff
@@ -25,19 +25,29 @@ else
     module load PrgEnv-gnu/6.0.4
 fi
 
-module load cudatoolkit/8.0.61_2.4.3-6.0.4.0_3.1__gb475d12
+module load daint-gpu
+# currently loads CUDA 8.0
+module load craype-accel-nvidia60
+
+# Compile for cluster nodes
+#   (CMake likes to unwrap the Cray wrappers)
+export CC=$(which cc)
+export CXX=$(which CC)
+
+# define cray compiler target architecture
+# if not defined the linker crashed because wrong from */lib instead
+# of */lib64 are used
+export CRAY_CPU_TARGET=x86-64
 
 # Libraries ###################################################################
+module load CMake/3.10.1
+
 module load cray-mpich/7.6.0
 module load cray-hdf5-parallel/1.10.0.3
 
-# Other Software ##############################################################
-#
-
-# Environment #################################################################
+# Self-Build Software #########################################################
 #
 # needs to be compiled by the user
-export CMAKE_ROOT=$SCRATCH/lib/cmake-3.10.1
 export BOOST_ROOT=$SCRATCH/lib/boost-1.62.0
```

Comment:
Maybe after `cscs.ch #30827` is resolved we can remove this one as well and just load a matching module :)